syntax = "proto3";

message Location {
  int32 id = 1;
  int32 person_id = 2;
  string longitude = 3;
  string latitude = 4;
  string creation_time = 5;
}

message LocationId {
  int32 id = 1;
}

service LocationService {
    rpc Create(Location) returns (Location);
    rpc Get(LocationId) returns (Location);
}